src/wp-content/themes/twentysixteen/inc/customizer.php

```diff
@@ -1158,14 +1148,9 @@ function twentysixteen_main_text_color_css() {
 		.comment-list .trackback,
 		.comment-reply-link,
 		.no-comments,
-		.widecolumn .mu_register .mu_alert {
```

Comment:
`.widecolumn .mu_register .mu_alert` and the border color were removed accidentally